.env.sample

```diff
@@ -43,16 +43,16 @@ ORTHANC_RAW_MAXIMUM_STORAGE_SIZE= // MB
 
 # PIXL Orthanc anon instance
 ORTHANC_ANON_USERNAME=
-ORTHANC_ANON_PASSWORD=     
+ORTHANC_ANON_PASSWORD=
 ORTHANC_ANON_AE_TITLE=
 ORTHANC_ANON_HTTP_TIMEOUT=60
-ORTHANC_AUTOROUTE_ANON_TO_AZURE=false
+ORTHANC_AUTOROUTE_ANON_TO_FTPS=true
```

Comment:
We want to actively choose to export data so I think false is a better default to minimise accidental data transfer during development

Maybe `ORTHANC_AUTOROUTE_ANON_TO_ENDPOINT` because we will choose to route to azure/DSH/another DICOMWeb endpoint

Reply:
Had a question about this during onboarding, but assuming this is set as default false how do we actually trigger the data to be sent from anon to FTPS currently?

Reply:
I assume by setting it locally (e.g. in `test/.env.test`) to `true`?

Reply:
> Had a question about this during onboarding, but assuming this is set as default false how do we actually trigger the data to be sent from anon to FTPS currently?

and likewise in production we change the .env to `true`

Reply:
Okay great, just wanted to clarify thanks